tests/Integration/PrestaShopBundle/ModuleFeatures/TemplatingInheritanceTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Integration\PrestaShopBundle\ModuleFeatures;
+
+use Tests\Integration\PrestaShopBundle\Test\WebTestCase;
+
+/**
+ * @group demo
+ */
+class TemplatingInheritanceTest extends WebTestCase
+{
+    /**
+     * @var string $moduleName The module name.
+     */
+    const MODULE_NAME = 'module_inheritance';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->installModule(self::MODULE_NAME);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function tearDown()
+    {
+        parent::tearDown();
+        $this->uninstallModule(self::MODULE_NAME);
+    }
+
+    /**
+     * @dataProvider getDataProvider
+     */
+    public function testTemplatesShouldHaveBeenOverridden($pageName, $pageTitle, $route)
```

Comment:
ping @jocel1 I may need help here: the test database have the module "module_inheritance" activated, but I can't manage to retrieve it in my container :/ I'm really close to finish this contribution, wdyt?

Reply:
Unit tests are not really unit, I maybe have a fix for this :)